hack/verify-lint.sh

```diff
@@ -30,5 +30,5 @@ for d in $(find . -type d -a \( -iwholename './pkg*' -o -iwholename './cmd*' \))
 		 --cyclo-over=60 \
 		 --dupl-threshold=100 \
 		 --tests \
-		 --deadline=300s "${d}"
+		 --deadline=600s "${d}"
```

Comment:
I had to go all the way to 900.. but really it's random issue based on memory use.. fix is in progress on gometalinter.. PR #289